spec/system/accessibility_spec.rb

```diff
@@ -88,9 +88,12 @@
 
   describe "Results page" do
     let(:estimate_id) { SecureRandom.uuid }
+    let(:calculation_result) do
+      CalculationResult.new(build(:api_result)).tap { _1.level_of_help = "certificated" }
+    end
```

Comment:
Does this (and the change below) mean that we don't have accessibility coverage for the Results page (controlled) or the print results page (certificated) ?

Reply:
Yes. I was being lazy. I will stop being lazy and add the missing tests.